### ISSUE TITLE
Agent name does not match reference used for relationship

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#5447]
Relevant to this: https://arctos.database.museum/agents.cfm?agent_id=21285400, relationship remarks contains a DOI.

Maybe not so relevant to this: There's a lot of info that doesn't agree between those resources, I'm not sure if I should believe it or not, I can't see the justification for that particular relationship, I can't find the Arctos agent from the name on the publication. We need better training/documetation, or this issue is still missing something, or ?????????

_Originally posted by @dustymc in https://github.com/ArctosDB/arctos/issues/5172#issuecomment-1373026910_

**Comment by Jegelewicz** [comment]
@jessicatir The name in the referenced publication is "Jack C. von Blo**e**ker, Jr." but the Arctos agent name is "J. C. von Blo**c**ker". Do you believe the Arctos agent name is misspelled? If so, I suggest that you add the verbatim agent attribute to the 4 affected records and use J. C. von Blo**c**ker as the attribute value, then change the Arctos agent to Jack C. von Blo**e**ker, Jr.

**Comment by Jegelewicz** [comment]
@ebraker can you confirm or deny any of this?

**Comment by jessicatir** [comment]
The specimen tag has "Jack C. von Blo**e**ker, Jr" so I do think it's misspelled on Arctos.

**Comment by Jegelewicz** [comment]
Also
[Jack C. Von Bloeker Jr.](https://arctos.database.museum/agents.cfm?agent_id=10001217)
[Jack von Bloeker](https://arctos.database.museum/agents.cfm?agent_id=21298286)
[O. B. Von Bloeker](https://arctos.database.museum/agents.cfm?agent_id=10012420)

**Comment by lin-fred** [comment]
Also here is more information about this person and his family https://www.islapedia.com/index.php?title=VON_BLOEKER,_Jack_C.,_Jr.

**Comment by dustymc** [comment]
@cjconroy do you know anything else about https://arctos.database.museum/agents.cfm?agent_id=10012420? JC seems to have a O.-something daughter.

**Comment by ebraker** [comment]
@Jegelewicz @jessicatir Yes, looks like the agent is misspelled - I see Jack C. von Bloeker, Jr. in the attached ledger photo. I'm fine with updating to Jack C. von Bloeker, Jr. (in this case I will not plan to add verbatim agents to UCM records since it is a transcription error and attached media confirm we've always had the full name - Jack C. von Bloeker, Jr. - on record, e.g. no inference is being made).

**Comment by Jegelewicz** [comment]
Need to remove any information from J. C. von Blocker and mark as bad duplicate of [Jack C. Von Bloeker Jr.](https://arctos.database.museum/agents.cfm?agent_id=10001217)

**Comment by Jegelewicz** [comment]
It looks like the other von Bloekers have been cleaned up too. Closing
